### ISSUE TITLE
The wrong error message is displayed when entering strings or special characters in the "Branch Size" section

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [mozilla/experimenter#990]
**[Affected versions]:**
- Firefox Release 65.0.2 and above
- Chrome 72.0.3626.96

**[Affected Platforms]:** 	 	
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have an experiment created on the "https://experimenter.stage.mozaws.net/" website.
- You are logged in your LDAP account.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites and navigate to "https://experimenter.stage.mozaws.net/".
2. Open the experiment from prerequisites and click the "Edit" button from "Branches" section.
3. Insert a string in the "Branch Size" section.
4. Click the "Save draft" button and observe the error message displayed under the "Branch Size" input field.

**[Expected result]:**
- An error message that implies that a whole number should be inserted is displayed.

**[Actual result]:**
- The "! The size of all branches must add up to 100." error message is displayed.

**[Notes]:**
- Attached is a screen recording of the issue:
![errormessage](https://user-images.githubusercontent.com/40061643/53636325-7c1acb80-3c28-11e9-91ed-4b2b33b29346.gif)

**Comment by peterbe** [comment]
Is this perhaps something that's been fixed already but not made it to Stage?

![screen shot 2019-03-01 at 11 41 20-fullpage](https://user-images.githubusercontent.com/26739/53652494-05290700-3c17-11e9-9e71-a25e8428647b.png)

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue on https://experimenter-app.dev.mozaws.net/ and it's not reproducible. Considering this I will close this issue as verified fix.

**Comment by Softvision-RemusDranca** [comment]
I have retested this issue and it's still reproducible on the "https://experimenter.stage.mozaws.net/" and also on the  "https://experimenter-app.dev.mozaws.net/"  website after the (v2.25.1) deployment.
This issue is also reproducible if numbers with decimals are inserted in the "Branch Size" input field.
Tested on Windows 10x64, MacOS 10.14 and Arch Linux 4.16.6 x64.
Considering this, I will reopen this issue and remove the [QA]:Verified fixed label.
